domains/certificates/Query.API/API/ContractService/ContractServiceImpl.cs

```diff
@@ -64,7 +59,7 @@ internal class ContractServiceImpl : IContractService
             contractNumber,
             gsrn,
             matchingMeteringPoint.GridArea,
-            MeteringPointType.Production,
+            meteringPointType,
```

Comment:
The meteringPointType can be determined by the matchingMeteringPoint.

Reply:
corrected